RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -29,4 +29,4 @@
                                                        872423215,872416066,872435036,872439336,
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
-                                                       872439603])
+                                                       872439603,872422436])
```

Comment:
this is a part of standard miniAOD filters.
The no-change policy requires that results of the default miniAOD setup are unchanged.

If you want to propose this change for analysis, it can be added with some instructions to edit this file or with a customization method or a process modifier.

If it is needed for a new central miniAOD reprocessing, then we need to define a new era for this reprocessing and add this detid there.

Reply:
Until plans for a new MiniAOD in 94X and a new era is defined, this backport may stay on the todo list (hold?) then.

Reply:
sure, we can do that.